R/conga.R

```diff
@@ -1,16 +1,18 @@
-#' Calculate continuous overall net glycemic action (CONGA)
+#' Continuous Overall Net Glycemic Action (CONGA)
 #'
 #' @description
 #' The function conga produces a CONGA values a tibble object.
-#' conga currently only supports calculation of CONGA24.
+#' conga now supports calculation of any n hours apart.
```

Comment:
I will probably take this out, this more of news (update compared to last change), and the part about any n hours should go in the description I think